src/patterns/components/dividers/base.hbs

```diff
@@ -3,8 +3,6 @@ name: Basic Divider
 order: 1
 hasAngularCode: true
 hasAngularCodeInfo: true
-hasReactCode: true
```

Comment:
Is this PR removing the divider?

Reply:
It shouldn't be removing it! There was some issues with dividers and merging staging into my branch. This may be residual from that. I'll take a look. There shouldn't be any changes to dividers in this pr.